clang/test/SemaSYCL/accessor-type-diagnostics.cpp

```diff
@@ -7,7 +7,7 @@
 using namespace cl::sycl;
 
 template <typename name, typename Func>
-__attribute__((sycl_kernel)) void kernel(Func kernelFunc) {
+__attribute__((sycl_kernel)) void kernel(const Func &kernelFunc) {
   kernelFunc();
 }
```

Comment:
I would remove this definition. The same template is defined in sycl.hpp included above.

Reply:
done